app/dashboard/helpers.py

```diff
@@ -336,6 +323,27 @@ def process_bounty_details(bountydetails, url, contract_address, network):
             for interested in last_bounty.interested.all():
                 new_bounty.interested.add(interested)
 
+        if fments:
+            for fment in fments:
+                new_fulfillment = BountyFulfillment.objects.create(
+                    fulfiller_address=fment.get('payload', {}).get('fulfiller', {}).get('address', '0x0000000000000000000000000000000000000000'),
+                    fulfiller_email=fment.get('payload', {}).get('fulfiller', {}).get('email', ''),
+                    fulfiller_github_username=fment.get('payload', {}).get('fulfiller', {}).get('githubUsername', ''),
+                    fulfiller_name=fment.get('payload', {}).get('fulfiller', {}).get('name', ''),
+                    fulfiller_metadata=fment.get('payload', {}).get('metadata', {}),
+                    bounty=new_bounty,
+                    )
+                new_fulfillment.save()
+
+            inactive_bounties = Bounty.objects.filter(
```

Comment:
why delete the inactive bounties?  IMHO these are useful to have for analytics purposes

Reply:
I'm not actually deleting the inactive bounties, I'm using the inactive bounties filter to delete the old fulfillment data.  No data actually gets deleted in the process, its just deleting duplicate data.  How it works is:

1.  There is an Existing Bounty
2.  That bounty gets fulfilled.
3 . "New Bounty" is created containing that data.  This also populates the BountyFulfillment table.
3.  Bounty gets fulfilled again.
4 . "New Bounty" is created containing that data.  This also populates the BountyFulfillment table.  But now we have a duplicate fulfillment that existed from the first case.
5.  Clean up duplicate bounties.

Reply:
ah you are right.. my bad